src/main/webapp/js/common.js

```diff
@@ -457,7 +457,7 @@ function getPointValue(s, ditchZero) {
     
     if (s0.indexOf('/') !== -1) {
         if (s0.indexOf('S') !== -1) {
-            return 999; // Case N/S
+            return 999; // Case N/S (feedback contribution not sure)
```

Comment:
999 and 1000 are arbitrary values that will encounter collisions. `Number.MAX_VALUE` and `Number.POSITIVE_INFINITY` should be used instead.